.github/workflows/README.md

```diff
@@ -0,0 +1,115 @@
+# Workflow Taxonomy
+
+## Where and what tests are run
+
+---
+
+### Workflow: `deploy-happy-stack.yml`
+
+#### Ran on:
+
+- `deployment`: (deployments are triggered by `push-tests`)
+
+#### Jobs:
+
+- ##### `upgrade`: TODO
+- ##### `e2e-test`:
+  - Runs playwright against deployed frontend app uploading any artifacts
+    - ```bash
+            if [ "${DEPLOYMENT_STAGE}" == "stage" ]; then
+            export DEPLOYMENT_STAGE=staging
+          fi
+          if [ "${DEPLOYMENT_STAGE}" != "prod" ]; then
+            pip3 install -r scripts/smoke_tests/requirements.txt
+            python3 -m scripts.smoke_tests.setup
+          fi
+          cd frontend
+          npm ci
+          npx playwright install --with-deps
+          cp src/configs/${DEPLOYMENT_STAGE}.js src/configs/configs.js
+          DEBUG=pw:api npm run e2e-${DEPLOYMENT_STAGE}
+      ```
+- ##### `functional-test`:
+  - If environment is not `prod` start up backend container and run `make local-functional-test`
```

Comment:
Am I crazy or does this not actually run in the container that's started up? https://github.com/chanzuckerberg/single-cell-data-portal/blob/ee3e590bab9fb77fd814c28df8a13fc4d38b1c83/.github/workflows/deploy-happy-stack.yml#L218-L219

Reply:
It does, the make command does some set-up and runs the actual tests in the docker container. 

I do wonder what the point of running this post-deploy is if it's being run against a docker container anyway, though...

Reply:
ah I misread the Dockerfile, was looking at the wrong target

Reply:
Huh yeah, could have been a copy/paste error. Probably should run functional test against the deployed BE 😆 Great catch!

Reply:
Might be good to include that implementation detail here for clarity

Reply:
Ah interesting! Would it make sense to drop `local-` in `local-functional-test`?

Reply:
Yeah, that's probably appropriate

Reply:
I'm not actually sure if dropping local here makes sense, since it runs the test in “local dev” based on the happy team's definition

Reply:
yeah I think I've changed my mind on this, ultimately we can obscure details like that as long as we do a good job illustrating the point here